app/assets/v2/js/pages/bounty_details.js

```diff
@@ -111,6 +111,16 @@ var callbacks = {
 
     if (ui_status == 'open') {
       ui_status = '<span>' + gettext('OPEN ISSUE') + '</span>';
+
+      let soft = result['can_submit_after_expiration_date'];
+
+      if (soft && is_bounty_expired(result)) {
+        ui_status += '<p class="text-highlight-light-blue" style="text-transform: none;">' +
+          gettext('This issue is past its expiration date, but it is still active.') +
+          '<br>' +
+          gettext('Check with the submitter to see if they still want to see it fulfilled.') +
```

Comment:
Should we update to leave a comment on the github issue to make it certain on how the should get in touch with the funder ?

@SaptakS

Reply:
@mbeacom

Reply:
i think its pretty clear how to get in touch with the funder from the gitcoin issue